include/core/CMemoryDef.h

```diff
@@ -489,8 +493,11 @@ void dynamicSize(const char* name,
     std::string componentName(name);
     componentName += "_umap";
 
-    std::size_t mapSize = (t.bucket_count() * sizeof(std::size_t) * 2) +
-                          (t.size() * (sizeof(K) + sizeof(V) + 2 * sizeof(std::size_t)));
+    std::size_t mapSize =
+        (t.bucket_count() * sizeof(std::size_t) * 2) +
+        (t.size() * (sizeof(K) + sizeof(V) + 2 * sizeof(std::size_t))) +
```

Comment:
Even though it's not a problem you introduced, I just noticed that this method is not using a helper function that the others are.

```suggestion
        (t.bucket_count() * memory::storageNodeOverhead(t)) +
        (t.size() * (sizeof(K) + sizeof(V) + memory::storageNodeOverhead(t))) +
```